### PR TITLE
fix(UiKit): Modal validation not working

### PR DESCRIPTION
**Pull request by tiagoevanp**
## Proposed changes (including videos or screenshots)

Bring back modal validations response to inputs:

image

## Further comments

[CORE-469](https://rocketchat.atlassian.net/browse/CORE-469?atlOrigin=eyJpIjoiMDJlMTA0MTc2ZjQwNDc4OWI2OWVjMWY5ZjU1YmRmZjIiLCJwIjoiaiJ9)

[CORE-469]: https://rocketchat.atlassian.net/browse/CORE-469?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]
Can we add tests to ensure it?

**Review by MarcosSpessatto — APPROVED** [review]

**Review by MarcosSpessatto — APPROVED** [review]

File: apps/meteor/tests/e2e/apps/apps-modal.spec.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+test.use({ storageState: Users.user1.state });
+
+test.describe.serial('Apps > ContextualBar', () => {
+	let poHomeChannel: HomeChannel;
+	let poModal: Modal;
+
+	let page: Page;
+
+	test.beforeAll(async ({ browser }) => {
+		page = await browser.newPage();
```
Comment (MartinSchoeler) [inline apps/meteor/tests/e2e/apps/apps-modal.spec.ts:17]: Close this page after all

File: apps/meteor/tests/e2e/apps/apps-contextualbar.spec.ts [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 
-test.describe.serial('Apps', () => {
+test.describe.serial('Apps > ContextualBar', () => {
 	let poHomeChannel: HomeChannel;
 
-	test.beforeEach(async ({ page }) => {
+	let page: Page;
+
+	test.beforeAll(async ({ browser }) => {
+		page = await browser.newPage();
```
Comment (MartinSchoeler) [inline apps/meteor/tests/e2e/apps/apps-contextualbar.spec.ts:15]: Close this page after all

File: apps/meteor/tests/e2e/page-objects/fragments/home-content.ts [hunk 3]
```diff
@@ -89,8 +89,7 @@ export class HomeContent {
 	async dispatchSlashCommand(text: string): Promise<void> {
 		await this.joinRoomIfNeeded();
 		await this.page.waitForSelector('[name="msg"]:not([disabled])');
-		await this.page.locator('[name="msg"]').fill('');
-		await this.page.locator('[name="msg"]').type(text);
+		await this.page.locator('[name="msg"]').fill(text);
```
Comment (MartinSchoeler) [inline apps/meteor/tests/e2e/page-objects/fragments/home-content.ts:93]: if these are not required in this PR, lets remove it and add in a new one

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]

**Review by MartinSchoeler — APPROVED** [review]

**Pull request merged**